### PR TITLE
Added

### PR DESCRIPTION
**Pull request by LeCloutPanda**
Added two nodes 
Sorted list of nodes alphabetically

### CODE REVIEW

File: Logix/Math/Random/RandomDouble.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+{
+    [Category("LogiX/Math/Random")]
+    [NodeName("Random Double")]
+    public class RandomDouble : LogixNode
+    {
+        public readonly Input<double> Min;
+        public readonly Input<double> Max;
+        public readonly Output<double> Value;
+
+        protected override void OnEvaluate()
```
Comment (Frozenreflex) [inline Logix/Math/Random/RandomDouble.cs:16]: Random should probably be abstracted elsewhere, we shouldn't be creating new random objects every time we run this node. This also needs the min max swap that the vanilla random nodes have
```cs
    protected override void OnEvaluate()
    {
      float min = Min.EvaluateRaw();
      float max = Max.EvaluateRaw();
      if (min > max)
      {
        float num1 = max;
        float num2 = min;
        min = num1;
        max = num2;
      }
      Value.Value = RandomX.Range(min, max);
    }
```

File: Logix/Users/IsUserInSeatedMode.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        public readonly Input<User> User;
+
+        public override bool Content
+        {
+            get
+            {
+                var user = User.EvaluateRaw();
+
+                try
```
Comment (Frozenreflex) [inline Logix/Users/IsUserInSeatedMode.cs:17]: This should be replaced with a null-coalescing operator
```cs
public override bool Content => User.EvaluateRaw()?.InputInterface.SeatedMode ?? false;
```

File: NEOSPlus.csproj [hunk 3]
```diff
@@ -41,6 +41,10 @@
     <Reference Include="BaseX">
       <HintPath>$(NeosPath)Neos_Data/Managed/BaseX.dll</HintPath>
     </Reference>
+    <Reference Include="CodeX, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (Frozenreflex) [inline NEOSPlus.csproj:44]: Remove the version text
```
    <Reference Include="CodeX">
      <HintPath>$(NeosPath)Neos_Data/Managed/CodeX.dll</HintPath>
    </Reference>
```

File: NEOSPlus.csproj [hunk 4]
```diff
@@ -89,19 +94,24 @@
...
     <Compile Include="Logix\Slots\GetGrandparent.cs" />
+    <Compile Include="Logix\Users\IsUserInSeatedMode.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Materials\TestMaterial.cs" />
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="SourceGenerators\SourceGenerators.csproj" OutputItemType="Analyzer" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
```
Comment (Frozenreflex) [inline NEOSPlus.csproj:112]: Remove this, your IDE injected the outdated format

File: app.config [hunk 5]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Frozenreflex) [inline app.config:1]: Delete this file

File: packages.config [hunk 6]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (Frozenreflex) [inline packages.config:1]: Delete this file

**Review by Frozenreflex — CHANGES_REQUESTED** [review]

**Review by LeCloutPanda — COMMENTED** [review]
Should all be fixed

**Review by Frozenreflex — APPROVED** [review]

**Pull request merged**